src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
+/* eslint-disable */
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function promise1() {
+  logo.addEventListener('click', () =>
+    new Promise((resolve) => resolve('Promise was resolved!')).then(
+      (msg) => (document.body.innerHTML += `<div class="message">${msg}</div>`),
```

Comment:
instead of adding inner HTML as string it's better to create a new div element, set the correct content and class for it, and append in the body

Such code would be easier to read and change (mantain)